less/forum/Post.less

```diff
@@ -116,6 +116,14 @@
     border-bottom: 2px dotted @body-bg;
     margin: 1em 0;
   }
+  code {
+    font-family: source-code-pro, Monaco, Consolas, "Courier New", monospace;
+    padding: 2px 6px;
+    background: #60646D;
+    color: #256FC7;
```

Comment:
@datitisev Maybe have a look on one of these variables, IMO these should look good.
https://github.com/datitisev/core/blob/814-better-post-code-style/less/lib/variables.less#L30-L31